schemas/v3.0/README.md

```diff
@@ -1,27 +1,40 @@
 # OpenAPI 3.0.X JSON Schema
 
-Here you can find the JSON Schema for validating OpenAPI definitions of versions 3.0.x.
+This directory contains the YAML source for generating the JSON Schema for validating OpenAPI definitions of versions 3.0.X, which is published on [https://spec.openapis.org](https://spec.openapis.org).
 
-As a reminder, the JSON Schema is not the source of truth for the Specification.
-In cases of conflicts between the Specification itself and the JSON Schema, the
-Specification wins. Also, some Specification constraints cannot be represented
-with the JSON Schema so it's highly recommended to employ other methods to
-ensure compliance.
+Due to limitations of GitHub pages, the schemas on the spec site are served with `Content-Type: application/octet-stream`, but should be interpreted as `application/schema+json`.
+
+The source in this directory, which has `WORK-IN-PROGRESS` in its `id`, is _not intended for direct use_.
+
+## Schema `id` dates
+
+The published schemas on the spec site have an _iteration date_ in their `id`s.
+This allows the schemas for a release line (in this case 3.0) to be updated independent of the spec patch release cycle.
 
 The iteration version of the JSON Schema can be found in the `id` field.
 For example, the value of `id: https://spec.openapis.org/oas/3.0/schema/2019-04-02` means this iteration was created on April 2nd, 2019.
 
-## Contributing
+The special URL with `latest` in place of the date is intended to provide an HTTP redirect to the most recent dated schema.
```

Comment:
```suggestion
<!-- The special URL with `latest` in place of the date is intended to provide an HTTP redirect to the most recent dated schema. -->
```
Has someone figured out how to do this with Jekyll?

https://stackoverflow.com/questions/39657226/jekyll-redirect-link may be the answer, if it also works with extension-less non-MD/HTML files.

Reply:
@ralfhandl it does not.  We cannot do a non-HTML-based redirect with GitHub pages.  I have been down this path before, and I tried it again yesterday.  GitHub pages is just inadequate for our needs and they have made clear statements in the past that they will never fix this because it's out-of-scope (as is proper setting of media types).

Reply:
Then we should remove this sentence unless/until we decide whether to have a `latest` or `latest.json` symlink.